### PR TITLE
multi: updates for release v0.8.5-alpha

### PR DESCRIPTION
**Pull request by ellemouton**
Bump the Loop version to v0.20.2-beta Pool version to v0.6.1-beta. 

We also bump the minimum compatible LND version to v0.15.4-beta.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +130,7 @@ The following table shows the supported combinations:
 
 | LiT              | LND          | Loop         | Faraday      | Pool         |
 |------------------|--------------|--------------|--------------|--------------|
+| **v0.8.4-alpha** | v0.15.5-beta | v0.20.2-beta | v0.2.9-alpha | v0.6.2-beta  |
```
Comment (guggero) [inline README.md:133]: nit: lit version not bumped. And Pool should be v0.6.1-beta.

**Review by guggero — APPROVED** [review]
Nice, thanks a lot :tada:

**Comment by ellemouton** [comment]
erg - did something wrong again with the commits 🙈

File: README.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ versioning](#daemon-versions-packaged-with-lit).
 
 | LiT              | LND          |
 |------------------|--------------|
+| **v0.8.4-alpha** | v0.15.4-beta |
```
Comment (ellemouton) [inline README.md:79]: erg

**Review by positiveblue — APPROVED** [review]
Thank you @ellemouton  :tada:

**Comment by ellemouton** [comment]
for now have just linked the commit hash with the pool fix just so I can see that the integration test passes. Will update to a tag once the pool release is out

**Comment by jamaljsr** [comment]
When creating a new account using the Pool UI in litd, I'm getting this error:
```
script mismatch: expected 00204d7e61e159a427f7b089632f165862bd466374198c50ebcadd83a2b6d76625c6, got 5120c8740787365259badea8c30b3c292832b5ca2a3ad2b362a9976e3c48f57febdd
```

litd logs:
```
zane              | 2022-12-02 16:54:44.582 [INF] ACCT: Creating new account 03a56dd849dbc9e1d7142717c0d1063637a8d81604f88ef69461979257e5910acc of 0.1 BTC that expires at height 13161
zane              | 2022-12-02 16:54:44.588 [INF] LNWL: Inserting unconfirmed transaction 976469b199067e971923f992611fa530acc6ed551a6a5f0cff7453901b521eb8
zane              | 2022-12-02 16:54:44.598 [INF] ACCT: Funded new account 03a56dd849dbc9e1d7142717c0d1063637a8d81604f88ef69461979257e5910acc with transaction 976469b199067e971923f992611fa530acc6ed551a6a5f0cff7453901b521eb8
zane              | 2022-12-02 16:54:44.610 [ERR] AUCT: [/poolrpc.ChannelAuctioneer/InitAccount]: rpc error: code = Unknown desc = script mismatch: expected 00204d7e61e159a427f7b089632f165862bd466374198c50ebcadd83a2b6d76625c6, got 5120c8740787365259badea8c30b3c292832b5ca2a3ad2b362a9976e3c48f57febdd
zane              | 2022-12-02 16:54:44.611 [ERR] RPCS: [/poolrpc.Trader/InitAccount]: rpc error: code = Unknown desc = script mismatch: expected 00204d7e61e159a427f7b089632f165862bd466374198c50ebcadd83a2b6d76625c6, got 5120c8740787365259badea8c30b3c292832b5ca2a3ad2b362a9976e3c48f57febdd
```

It does look like the account was created when I run `pool accounts list`, but it remains in the pending state even after the txn is confirmed
```
zane:/# pool --rpcserver=localhost:10009 --tlscertpath=/root/.lnd/tls.cert  --macaroonpath=/root/.pool/regtest/pool.macaroon accounts list
{
	"accounts": [
		{
			"trader_key": "03a56dd849dbc9e1d7142717c0d1063637a8d81604f88ef69461979257e5910acc",
			"outpoint": "976469b199067e971923f992611fa530acc6ed551a6a5f0cff7453901b521eb8:1",
			"value": 10000000,
			"available_balance": 10000000,
			"expiration_height": 13161,
			"state": "PENDING_OPEN",
			"latest_txid": "976469b199067e971923f992611fa530acc6ed551a6a5f0cff7453901b521eb8",
			"version": "ACCOUNT_VERSION_TAPROOT"
		}
	]
}
```

```
zane              | 2022-12-02 17:00:07.540 [INF] LNWL: Marking unconfirmed transaction 976469b199067e971923f992611fa530acc6ed551a6a5f0cff7453901b521eb8 mined in block 202
```

I also noticed I run into this same issue when creating an account via the `pool` CLI. 
```
zane:/#  pool --rpcserver=localhost:10009 --tlscertpath=/root/.lnd/tls.cert  --macaroonpath=/root/.pool/regtest/pool.macaroon accounts new --amt=1000000000 --expiry_blocks=8064 --conf_target=6 --force
[pool] rpc error: code = Unknown desc = script mismatch: expected 002063f971b2da6a30752d04b4bf25c8e797c257c8411c059b0f601125ff5c9896f2, got 51205e15127383844bad56c3f7264f1d825396e4df565d9f5656130d573fe9fdac6f
```

**Comment by ellemouton** [comment]
thanks for spotting this @jamaljsr ! I will reach out to the loop team

**Pull request merged**